src/components/creatEventPageComponents/CreateEvent.jsx

```diff
@@ -0,0 +1,154 @@
+import { addDoc } from "firebase/firestore";
+
+import { eventsCollection } from "@/lib/firebase/controller";
+
+function CreateEvent() {
+    function handleCreateForm(e) {
+        e.preventDefault();
+
+        addDoc(eventsCollection, {
+            title: e.target.title.value,
+            about: e.target.about.value,
+            date: e.target.date.value,
+            image: e.target.image.value,
+            location: e.target.location.value,
+            category: e.target.category.value,
+        }).then(() => {
+            e.target.reset();
+        });
+    }
+
+    return (
+        <div className='bg-white border rounded-lg shadow relative m-10'>
+            <div className='flex items-start justify-between p-5 border-b rounded-t '>
+                <h3 className='text-2xl font-semibold'>Create Event</h3>
+                <button
+                    type='button'
+                    className='text-gray-600 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm p-1.5 ml-auto inline-flex items-center'
+                    data-modal-toggle='Event-modal'
+                >
+                    <svg
+                        className='w-5 h-5'
+                        fill='currentColor'
+                        viewBox='0 0 20 20'
+                        xmlns='http://www.w3.org/2000/svg'
+                    >
+                        <path
+                            fillRule='evenodd'
+                            d='M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z'
+                            clipRule='evenodd'
+                        ></path>
```

Comment:
same as the svg not sure what this path is for

Reply:
i totally missed it, it's feature of the template provided, which definetly can be ignored for our purpose of usage, thank you for this deep observation